notifications/05-05-1700.md

```diff
@@ -0,0 +1,31 @@
+
+**What is happening?**
+
+RFC 193772 - CITZ - PROD MCS - Add CPU and RAM resource reservations on all nodes
+
+We will be applying some recommended reservation of CPU and RAM resources on all PROD nodes as a means of improving system stability, by recommendation by Red Hat. 
+
+**When?**
+
+Next Tuesday evening (May 5th 2020) starting after 5:00pm PDT.
+
+**Will there be an impact on the Platform apps?**
+
+There is a theoretical risk where one or more pods on any one node might need to evacuate if there is an over-subscription of resources after putting in the resource reservation. Additional steps are being taken so that this change only happens on a single node at a time instead of it possibly happening on all nodes at once as what otherwise would happen when making a configuration change to the node group configurations.
+
+For applications utilizing best practices and leveraging scaled-up pods for hosting apps/services, there should be no possible impact due to the involved work. Applications not yet scaled up however might experience a small impact as single pod workloads, if designated by the scheduler as needing to be re-scheduled due to the new resource reservation in place.
+
+As each node is worked on, we will be tracking the net removal of pods and posting in #devops-alerts when we are done - the pods might also be due to regular activities taking place instead of our work.
+
+**Do I need to do anything?**
+
+No. The cluster will remain operational during the course of this change.
+
+**Check the #devops-alert channel for the announcement of when the change is complete and check the health of your app.**
+
+**Where do I get help if my app does not work after the change is complete?**
+
+Each Platform application has an assigned DevOps Specialist within the Ministry so contact them first. If you do not know who your assigned DevOps Specialist, check with the app's Product Owner.
+
+The DevOps Specialist will troubleshoot the issue with the app and if they need help, they will reach out to the Platform Services Team and the Developer Community in Rocket.Chat as per these [RocketChat Channel Use Guidelines](
+https://developer.gov.bc.ca/Getting-human-support-for-issues-not-covered-by-devops-requests).
```

Comment:
```suggestion
https://developer.gov.bc.ca/Getting-human-support-for-issues-not-covered-by-devops-requests).
```
The [Post-Outage Recovery Checklist](https://developer.gov.bc.ca/Post-Outage-Application-Health-Checklist) contains useful information to help ensure your app has recovered successfully after an outage.